### ISSUE TITLE
Question: fixed node

### ISSUE THREAD

**Issue by krvajal** [kieler/elkjs#212]
Is it possible to have a node with a fixed position and allow the rest to be layout around that position?

**Comment by soerendomroes** [comment]
No, this is not possible but you could do that yourself by moving the whole drawing based on the position of your reference node.
